tianshou/data/buffer/base.py

```diff
@@ -384,9 +389,10 @@ def __getitem__(self, index: Union[slice, int, List[int], np.ndarray]) -> Batch:
             "done": self.done[indices],
             "obs_next": obs_next,
             "info": self.get(indices, "info", Batch()),
+            # TODO: what's the use of this key?
```

Comment:
it's for policy's forward function extra return

Reply:
Is it ever used in the code-base? I find it hard to understand what this entry represents and why it is in this base building block of tianshou